ext/tags.go

```diff
@@ -83,18 +96,67 @@ var (
 	HTTPStatusCode = uint16TagName("http.status_code")
 
 	//////////////////////////////////////////////////////////////////////
+	// DB Tags
+	//////////////////////////////////////////////////////////////////////
+
+	// DBInstance is database instance name.
+	DBInstance = stringTagName("db.instance")
+
+	// DBStatement is a database statement for the given database type.
+	DBStatement = stringTagName("db.statement")
+
+	// DBType is a database type.
+	DBType = stringTagName("db.type")
+
+	// DBUser is a username for accessing database.
+	DBUser = stringTagName("db.user")
+
+	//////////////////////////////////////////////////////////////////////
+	// Message Bus Tag
+	//////////////////////////////////////////////////////////////////////
+
+	// MessageBusDestination is an address at which messages can be exchanged
+	MessageBusDestination = stringTagName("message_bus.destination")
+
+	//////////////////////////////////////////////////////////////////////
 	// Error Tag
 	//////////////////////////////////////////////////////////////////////
 
 	// Error indicates that operation represented by the span resulted in an error.
 	Error = boolTagName("error")
+
+	//////////////////////////////////////////////////////////////////////
+	// Log Tags
+	//////////////////////////////////////////////////////////////////////
+
+	// LogErrorKind is the type or kind of error.
+	LogErrorKind = stringTagName("error.kind")
```

Comment:
in OT-Go, we would use `Field` to represent these logging concepts, not tags... let me know if you need more explanation / direction.